.github/workflows/release.yaml

```diff
@@ -88,3 +89,25 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Build container images
+        id: build_container_images
+        run: |
+            # Build podman images locally
+            image_tag=${{ steps.get_tag.outputs.release_version }}
+            podman build \
+              -t quay.io/redhat-certification/chart-verifier:$image_tag \
+              -t quay.io/redhat-certification/chart-verifier:latest \
+              -t quay.io/redhat-certification/chart-verifier:main .
+
+      - name: Push to quay.io
+        id: push_to_quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: chart-verifier
+          tags: |
+            ${{ steps.get_tag.outputs.release_version }}
+            latest
+            main
```

Comment:
This syntax looks strange. The doc does seem to imply you can specify multiple tags separated by whitespace, so I'm sure this works, but wow it looks odd! (Feel free to resolve this, as there's no action I'm really looking to have done here)

Reply:
Actually, I lied - I'm pretty sure we don't want to tag main here. Maybe latest. This is a release workflow that triggers on tags. Main and often will contain commits that are beyond a given release.

Reply:
Sorry about the syntax :upside_down_face: I agree it's not super pretty, but I'd say I've seen worse.

You're right, I'll remove building and pushing of `main`. I'm wondering if we should keep the Build Trigger or create a new Github Workflow to build the `main` image.

To clarify, we have 4 types of images:
- `x.y.z` (i.e. version tags): built as part of the release process (currently using Quay Build Triggers, replaced in this PR by a step in "release.yaml" worfklow)
- `latest`: follows the latest `x.y.z`. Currently retagged at the end of the "build.yaml" on release, moved in this PR to "release.yaml"
- `0.1.0` (i.e. dev release): built on each push to the main branch, currently in "dev_release.yaml" workflow. I think we should leave this untouched.
-  `main`: built for each push on the main branch via a Quay Build Trigger.

So, AFAICT the `0.1.0` and the `main` tag are always in sync. How do we want to build `main` ? We have following options:
1. Leave the current Build Trigger untouched.
2. Create a new GitHub workflow specifically for building the `main` image (triggered on each push to the main branch)
3. Reuse the "dev_release.yaml" workflow as it is triggered for each push to the main branch already, and integrate building the main image there.

Not a big fan of 3. personally, as I prefer to separate concerns. 1. means that we keep a build trigger active, and I was hoping to get rid of all Build Triggers here (not that I dislike Build Triggers, but I'd prefer having all the builds happening in the same place i.e. GitHub Actions). That leaves us with 2., even though I'd prefer not to create yet another workflow, I believe that's reasonable.

Reply:
Option 2 or 3 is the right way forward. We don't have to make them the same workflow, we can just make the existing workflow for 0.1.0 reusable. Easy enough!

Reply:
Clarity: Dealing with the main tag doesn't need to happen in this PR. We can do it at a later time. The build trigger won't go away until this lands anyway.

Reply:
I added a workflow for tagging the main branch in this PR.